Makefile

```
@@ -312,6 +312,7 @@ bundle-clean: ## Clean generated bundle files
 
 .PHONY: bin-clean
 bin-clean: ## Clean downloaded binaries
+	@[ -d bin ] && chmod -R u+rwx bin || true
```

I don't quite understand how `chmod` would succeed in a scenario where `rm -f -r` would fail... what was the problem ?

---

Permissions in bin/k8s/1.24.2-linux-amd64 were readonly:
dr-xr-xr-x. 2 tbuskey tbuskey 4096 Jul  3 13:07 1.24.2-linux-amd64

---

This is the result of `make test` failing and not fixing the mode as per bef333e. It means that the SKIP_TESTS approach isn't a good workaround for the broken unit tests. Let's revert 48942c6 and ignore errors at https://github.com/openshift/sandboxed-containers-operator/blob/devel/Makefile#L128 instead. I'll create a PR for this.

---

Now that #429 is merged, please rebase and you can drop this commit.